### DISCUSSION TITLE
"Share PocketBase dashboard"

### DISCUSSION THREAD

**Discussion by bridevmx** [pocketbase/pocketbase#4537] [Q&A]
My idea is to share the PocketBase dashboard with an editor so that they can only modify the records. My question is whether the dashboard template is editable? Or how can I achieve my goal without reinventing the wheel and rewriting the entire dashboard?

**Comment by ganigeorgiev** [comment]
There is no support for admin roles at the moment and the UI is not customizable (this will change with https://github.com/pocketbase/pocketbase/discussions/4355).
